src/meta/test/ProcessorTest.cpp

```diff
@@ -2265,8 +2266,9 @@ TEST(ProcessorTest, TagIndexTest) {
         cpp2::CreateTagIndexReq req;
         req.set_space_id(1);
         req.set_tag_name("tag_0");
-        std::vector<std::string> fields{"tag_0_col_0"};
-        req.set_fields(std::move(fields));
+        cpp2::IndexFieldDef field;
+        field.set_name("tag_0_col_0");
+        req.set_fields({field});
```

Comment:
why don't use move ?

Reply:
In this case, it works either way.Memory will be destroyed when this code block is complete.